.vscode/settings.json

```
@@ -0,0 +1,7 @@
+{
+  "eslint.workingDirectories": [
+    "./packages/ui-components",
+    "./packages/web-components",
+  ],
+  "prettier.prettierPath": "./node_modules/prettier"
```

please leave a link to the bug report so we can remove this when it's resolved.

VS Code settings files are JSONC so it could be,
```suggestion
  // UNTIL https://github.com/prettier/prettier-vscode/issues/3019
  "prettier.prettierPath": "./node_modules/prettier"
```